lib/pipenv.js

```diff
@@ -30,13 +30,12 @@ async function pipfileToRequirements() {
   try {
     let res;
     try {
-      res = await spawn(
-        'pipenv',
-        ['lock', '--requirements', '--keep-outdated'],
-        {
-          cwd: this.servicePath,
-        }
-      );
+      res = await spawn('pipenv', ['lock', '--keep-outdated'], {
```

Comment:
This feels to me like a change that might be potentially breaking for people on some older `pipenv` versions. Do you know if there's a minimal `pipenv` version needed for these two commands?

Reply:
It's definitely breaking for some older versions as there was no `requirements` command. Could you please verify which version of `pipenv` is now required? We should specify it in docs